src/cards.py

```diff
@@ -0,0 +1,29 @@
+
+
+def sort_cards(cards):
+    """Sort a deck of cards."""
+    deck = []
+    new_deck = []
+    for card in cards:
+        if card.upper() == 'A':
+            deck.append((0, 'A'))
+
+        elif card.upper() == 'T':
+            deck.append((10, 'T'))
+
+        elif card.upper() == 'J':
+            deck.append((11, 'J'))
+
+        elif card.upper() == 'Q':
+            deck.append((12, 'Q'))
+
+        elif card.upper() == 'K':
+            deck.append((13, 'K'))
+
+        else:
+            deck.append((int(card), card))
+    deck.sort()
+    for card in deck:
+        card = card[1]
```

Comment:
why are you reassigning `card` here? You're just going to append to a new list anyway and you're not doing anything with the resulting card. You might as well just do

```python
new_deck.append(card[1])
```